libraries/bot-azure/src/test/java/com/microsoft/bot/azure/AzureQueueTests.java

```diff
@@ -41,7 +41,7 @@
 
 public class AzureQueueTests {
     private static final Integer DEFAULT_DELAY = 2000;
-    private static Boolean EMULATOR_IS_RUNNING = false;
+    private static Boolean emulatorIsRunning = false;
```

Comment:
```suggestion
    private static boolean emulatorIsRunning = false;
```

Just for parity with Tracy